spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/config/secretsmanager/SecretsManagerConfigDataLoaderIntegrationTests.java

```diff
@@ -117,6 +144,17 @@ void resolvesPropertyFromSecretsManager_PlainTextSecret_endingWithSlash() {
 		}
 	}
 
+	@Test
+	void resolvesPropertyFromSecretsManager_PlainTextSecret_SecretBinary() {
+		SpringApplication application = new SpringApplication(App.class);
+		application.setWebApplicationType(WebApplicationType.NONE);
+
+		try (ConfigurableApplicationContext context = runApplication(application,
+				"aws-secretsmanager:/blob/byte_certificate")) {
+			assertThat(context.getEnvironment().getProperty("byte_certificate")).isEqualTo("77,121,32,99,101,114,116,105,102,105,99,97,116,101");
```

Comment:
we should rather take the same string as where secrets is created and call `getBytes` on it rather than comparing strings in such a weird way